BE/src/main/java/com/example/User/User.java

```diff
@@ -31,10 +32,14 @@ public class User implements UserDetails {
     @Column(nullable = false)
     private String password;
 
+    @Column(nullable = false)
+    private Integer energy;
+
```

Comment:
Prefer Jakarta annotations over Hibernate. Also, set the min and max bounds
```suggestion
    @Nullable
    @Min(0) @Max(10)
    private Integer energy;
```